.github/workflows/linting-reusable.yml

```diff
@@ -0,0 +1,54 @@
+name: Reusable linting workflow
```

Comment:
I wish there was a way to hide reusables from the GH UI.

Reply:
for that we should create a `n8n-io/actions` repo, and start moving all re-usable actions/workflows in there.